packages/elements/src/pagination/index.ts

```diff
@@ -299,10 +147,17 @@ export class Pagination extends BasicElement {
   }
 
   /**
-   * Getter for text field as input part
+   * Reference input element
    */
-  @query('#input')
-  private input!: HTMLInputElement;
+  protected inputRef: Ref<HTMLInputElement> = createRef();
+
+  /**
+   * Getter for input element
+   */
+  protected get input (): HTMLInputElement {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    return this.inputRef.value!;
```

Comment:
It is not always defined (e.g. if you call the function before `firstUpdate`), thus it should return `HTMLInputElement | null` and check if inputRef is defined. As well, please rename the function to `inputElement` to be consistent with other elements.

Reply:
Fixed, Thanks